client-v2/src/util/pollingConfig.ts

```diff
@@ -14,12 +13,10 @@ export default (store: Store) =>
     if ((window as any).IS_INTEGRATION) {
       return;
     }
-    const path = selectV2SubPath(store.getState());
-    const match = matchFrontsEditPath(path) || matchIssuePath(path);
-    if (!match || !match.params.priority) {
+
+    const priority = selectPriority(store.getState());
```

Comment:
A single selector is a little cleaner, and lets us re-use the logic.

Reply:
@jennygrahamjones @ajwl this may affect your sparklines work (hopefully it's handy though!)